web/yo/app/scripts/services/mainutils.js

```diff
@@ -664,6 +664,125 @@ angular.module('oncokbApp')
                 return item.indexOf("$") !== 0;
             });
         }
+        function checkDifferenceBetweenTherapies(oldContent, newContent){
```

Comment:
If you want to make this a generic method and export, you probably should accept two arrays instead of two string. I don't see much of the use case here by exporting the method if the input needs to be therapy string format.

Reply:
This function is used in three different places. If I change it to arrays,  I should preprocess Strings in these three different places. And 4 more lines of similar codes are needed.

Reply:
Don't export the function if that's the case since it's highly customized.

Reply:
You can export a generic one and create a wrapper on top for local usage.